docs/index.md

```diff
@@ -0,0 +1,169 @@
+(index)=
+(guides)=
+(howtos)=
+(tutorials)=
+
+# CrateDB Guides and Tutorials
+
+CrateDB is a distributed and scalable SQL database for storing and analyzing
+massive amounts of data in near real-time, even with complex queries. It is
+PostgreSQL-compatible, and based on Lucene. 
+
+This section of the documentation guides you how to use CrateDB in practice.
+
+
+::::{grid} 3
+:padding: 0
+
+
+:::{grid-item-card} Installation
+:link: install
+:link-type: ref
+:link-alt: Installing CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`download_for_offline;1.3em`
+:::
+
+
+:::{grid-item-card} Getting Started
+:link: getting-started
+:link-type: ref
+:link-alt: Getting started with CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`rocket_launch;1.3em`
+:::
```

Comment:
Maybe the 🚀 is too much here? Do you have any other suggestions?

Icons can be selected from https://fonts.google.com/icons.

Reply:
Let's just make sure they are free to use :)